.buildkite/pipeline.yml

```diff
@@ -71,3 +71,10 @@ steps:
     depends_on:
       - step: "tests"
         allow_failure: false
+
+  - label: "Upload the fleet-server-package-mbp pipeline"
+    command: "buildkite-agent pipeline upload .buildkite/pipeline.package.mbp.yml"
+#    if:  "build.env('BUILDKITE_BRANCH') != null && build.env('BUILDKITE_TAG') == null && build.env('BUILDKITE_PULL_REQUEST') != null"
+    depends_on:
+      - step: "publish"
+        allow_failure: false
```

Comment:
Could it be used a trigger step instead of the uploading the pipeline ?

https://buildkite.com/docs/pipelines/trigger-step